### PR TITLE
feat/new page atlante, ready for content

### PR DESCRIPTION
**Pull request by Fupete**
@danielaiozzo @bfabio just created a new project page with random content, to serve as sandbox for our content designers for the new page about the Atlante... this is a PR draft

### CODE REVIEW

**Comment by danielaiozzo** [comment]
awesome

**Comment by Fupete** [comment]
@gianvitofanelli can you make tomorrow a PR targeting this branch to fix these three wrong urls? Tx

immagine

**Comment by Fupete** [comment]
@danielaiozzo @bfabio can u review the changes asap? Tx

**Comment by Fupete** [comment]
@bfabio @danielaiozzo draft again, waiting for a new version of PDFs and confirmation about the licenses. Content stuff. @gianvitofanelli will manage a couple of more commits. 

Please however check everithing else is ok... that we have to merge this today...

**Review by danielaiozzo — APPROVED** [review]

File: _projects/it/atlante-della-trasformazione-digitale.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+
+#### Definizione del *brief*
+Insieme ai coordinatori del corso, abbiamo stabilito il focus del progetto di ricerca, il suo perimetro, le attività e gli output attesi. Inoltre, abbiamo condiviso con gli studenti non solo un *framework* di riferimento per analizzare il contesto, ma anche una serie di strumenti pratici per supportare la progettazione.
+
+#### Workshop 1: analisi del contesto
+Abbiamo organizzato una sessione di un giorno per aiutare gli studenti a prendere familiarità con il contesto e gli ambiti che avrebbero investigato e a creare un *framework* di riferimento per raccogliere e analizzare le informazioni. Grazie agli output di questo *workshop*, gli studenti hanno potuto analizzare il contesto dei servizi pubblici digitali rispetto a diversi ambiti della vita civica, identificando attori ed elementi coinvolti e le loro interrelazioni, per mappare l’ecosistema complessivo e definire i diversi archetipi delle organizzazioni e degli *stakeholder* coinvolti.
```
Comment (danielaiozzo) [inline _projects/it/atlante-della-trasformazione-digitale.md:84]:
```suggestion
Abbiamo organizzato una sessione di un giorno per aiutare gli studenti a prendere familiarità con il contesto e gli ambiti che avrebbero investigato e per creare un *framework* di riferimento per raccogliere e analizzare le informazioni. Grazie agli output di questo *workshop*, gli studenti hanno potuto analizzare il contesto dei servizi pubblici digitali rispetto a diversi ambiti della vita civica, identificando attori ed elementi coinvolti e le loro interrelazioni, per mappare l’ecosistema complessivo e definire i diversi archetipi delle organizzazioni e degli *stakeholder* coinvolti.
```

**Comment by Fupete** [comment]
@bfabio almost ready, i'm pushing a huge new feature with all the pdf renewed... wait for confirmation.

**Comment by Fupete** [comment]
@bfabio I think we are ready. Let's wait for ci/circleci tests... and let's do a final round of checks before merge.

**Comment by Fupete** [comment]
@bfabio ready to merge, as u wish

**Pull request merged**